packages/hebao_v2/contracts/base/WalletFactory.sol

```diff
@@ -32,6 +32,7 @@ contract WalletFactory is WalletDeployer
         address   inheritor;
         address   feeRecipient;
         address   feeToken;
+        uint      maxFeeAmount;
```

Comment:
I suggest that we remove `feeAmount` from this struct. And pass in feeAmount to `createWallet` as a seperate parameter directly:

```solidity
 function createWallet(
        WalletConfig calldata config,
        uint                  salt,
        uint                  feeAmount
        )

```